projects/demo-playwright/tests/addon-table/table-pagination/table-pagination.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, Page, test} from '@playwright/test';
+
+test.describe(`TablePagination`, () => {
+    test.describe(`Dropdown with \`[size]\`-options (amount items per page)`, () => {
+        test(`Basic case`, async ({page}) => {
+            await tuiGoto(page, `/components/table-pagination/API`);
+            const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+            const {tablePagination, linesPerPageSelect} =
+                await initializeAliases(apiPageExample);
```

Comment:
Look into other playwright tests.
They use Page object pattern.
Let's create `TuiTablePaginationPO`:

```ts
const tablePagination = new TuiTablePaginationPO(apiPageExample.locator(`tui-table-pagination`));

// [...]

await tablePagination.linesPerPageSelect.click()
```

Reply:
fixed in 1a7f7e6d3d63c38ad73aa828bbd637e8afb0dcb4